lib/schedule/trip.ex

```diff
@@ -23,7 +23,9 @@ defmodule Schedule.Trip do
           run_id: Run.id() | nil,
           stop_times: [StopTime.t()],
           start_time: Util.Time.time_of_day(),
-          end_time: Util.Time.time_of_day()
+          end_time: Util.Time.time_of_day(),
+          start_place: String.t(),
+          end_place: String.t()
```

Comment:
The implementation says this will be `nil` if we don't have it. (The default values in `defstruct` could also be changed to `nil`.)